app/src/main/java/com/googlecreativelab/drawar/DrawAR.java

```diff
@@ -527,7 +529,7 @@ public void onDrawFrame(GL10 gl) {
         mBackgroundRenderer.draw(mFrame);
 
         // Draw Lines
-        if (mFrame.getCamera().getTrackingState() == TrackingState.TRACKING) {
+        if (mFrame != null && mFrame.getCamera().getTrackingState() == TrackingState.TRACKING) {
```

Comment:
I don't think this will prove to be helpful because if in case `mFrame` is `null`, the condition will still go false without notifying the developer about it.